src/core/api/dataset_ftd_api.cpp

```diff
@@ -87,14 +89,15 @@ otError otDatasetSendMgmtActiveSet(otInstance *                aInstance,
     return instance.GetThreadNetif().GetActiveDataset().SendSetRequest(*aDataset, aTlvs, aLength);
 }
 
-otError otDatasetSendMgmtPendingGet(otInstance *        aInstance,
-                                    const uint8_t *     aTlvTypes,
-                                    uint8_t             aLength,
-                                    const otIp6Address *aAddress)
+otError otDatasetSendMgmtPendingGet(otInstance *                          aInstance,
+                                    const otOperationalDatasetComponents *aDatasetComponents,
+                                    const uint8_t *                       aTlvs,
```

Comment:
nit: `aTlvs` -> `aTlvTypes`

Reply:
@jwhui coming to change this.... looks you already fix this... Thanks :)